bedrock/firefox/templates/firefox/accounts-2019.html

```diff
@@ -0,0 +1,154 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros-protocol.html" import feature_card with context %}
+{% from "macros.html" import fxa_email_form, fxa_link_fragment with context %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{%- block page_title -%}
+  {# L10n: HTML page title #}
+  {{ _('Get a Firefox Account – Keep your data private, safe and synced') }}
+{%- endblock -%}
+{%- block page_desc -%}
+  {# L10n: HTML page description #}
+  {{ _('Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One login – Power and privacy everywhere.') }}
+{%- endblock -%}
+
+{% block page_css %}
+ {{ css_bundle('firefox_accounts_2019') }}
+{% endblock %}
+
+{% block body_class %}firefox-accounts mzp-t-firefox state-fxa-default {% endblock %}
+
+{% block site_header %}
+  {% with hide_nav_download_button=True %}
+    {% include 'includes/protocol/navigation/index.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block content %}
+
+
+<section class="c-accounts-hero">
+  <div class="mzp-l-content">
+    <div class="c-accounts-hero-body">
+      {# L10n: the span here is for visual formatting to display the phrase "Join Firefox" in a different color. #}
+      <h1 class="c-accounts-hero-title">{{ _('There is a way to protect your privacy. <span>Join Firefox.</span>') }}</h1>
+
+      {# L10n: This refers to the way some tech companies treat personal information as a commodity, as if their users are the product they're selling to their advertisers. #}
+      <p class="c-accounts-hero-desc">{{ _('Take your stand against an industry that’s making you the product.') }}</p>
+    </div>
+
+    <div class="c-accounts-form">
+      {{ fxa_email_form(
+        entrypoint='mozilla.org-accounts_page',
+        all_browsers=True,
```

Comment:
Does this mean we're no longer signing Firefox users into Sync as well?

Reply:
oh... damn, you're right. Hmm... ok, need to rethink this.